### PR TITLE
[ARM/CI] Enable Ubuntu 16.04 ARM CI build

### PR DESCRIPTION
**Pull request by jyoungyun**
Dummy PR for checking Ubuntu 16.04 ARM CI build

Related issue: #1577

### CODE REVIEW

**Comment by jyoungyun** [comment]
@dotnet-bot test ci please

**Comment by jyoungyun** [comment]
@dotnet-bot help
@dotnet-build-bot help

**Comment by jyoungyun** [comment]
@dotnet-bot test ci please

**Comment by jyoungyun** [comment]
@gkhanna79 Do you know what function register a new CI job in netci.groovy ? Currently there is no add CI job for Ubuntu 16.04 arm so I have to make another PR for fixing netci.groovy. If you can understand that a new CI job is broken every time a PR is generated, I would like to test it out by removing the exception handling syntax below. Is it possible?
https://github.com/dotnet/core-setup/blob/master/netci.groovy#L87L92

**Comment by gkhanna79** [comment]
@mmitche should be able to tell. In general, I would expect that the CI test job generated by _test ci please_ should have generated the Ubuntu 16.04 arm job entry when you click the _Details_ link. But I do not see any such jobs.

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
The netci.groovy changes are not live to this PR.  They are only live after commit.  I know this is not ideal, but that is the way it works today.  My suggestion here to make a PR that enables Ubuntu 16.04 (like you did in https://github.com/dotnet/core-setup/blob/master/netci.groovy#L87L92 but enables it only with a trigger that doesn't run by default on all PRs.  Then your PR to fix Ubuntu 16.04 can both update the code to make it pass as well as enable the trigger by default.  When you commit, 16.04 will become default and will work.

**Comment by mmitche** [comment]
@jyoungyun Hang on I'll bring up a branch that does what you want.

**Comment by mmitche** [comment]
@dotnet-bot help

**Comment by mmitche** [comment]
@dotnet-bot test Ubuntu16.04 arm Release Build

**Comment by jyoungyun** [comment]
@mmitche Thank you very much! I think I can't use `latest-or-auto-docker` image in `Ubuntu 16.04` for building CI job due to lack of `/opt/linux-arm-emulator/platform/rootfs-u1404.ext4`. Can I use `arm-cross-latest` in `ubuntu 14.04` image here temporarily ? Or is there anyone to add those ext4 file system into `Ubuntu 16.04` image like `arm-cross-latest`?

**Comment by jyoungyun** [comment]
@dotnet-bot test Ubuntu16.04 arm Release Build

**Comment by jyoungyun** [comment]
@dotnet-bot test Ubuntu16.04 arm Release Build

**Comment by jyoungyun** [comment]
@dotnet-bot test Ubuntu16.04 arm Release Build Build

**Comment by jyoungyun** [comment]
This failure seems to be related to Ubuntu server status. I will check it out next monday.

**Comment by mmitche** [comment]
@jyoungyun I actually saw this in another place.  I'll look into it and see what I find later.

**Comment by gkhanna79** [comment]
@jyoungyun Were you able to look into this?

**Comment by jyoungyun** [comment]
@gkhanna79 I'm trying to build rootfs for ubuntu xenial, but it is still unstable on Ubuntu server. I will ping you if this is ready to merge.

**Comment by gkhanna79** [comment]
Can you use the new v2 docker images we are now using that contain rootfs?



Thanks,
Gaurav


-------- Original message --------

**Comment by jyoungyun** [comment]
@dotnet-bot test Ubuntu16.04 arm Release Build

**Comment by jyoungyun** [comment]
@dotnet-bot test Ubuntu16.04 arm Release Build

**Comment by jyoungyun** [comment]
@dotnet-bot test Ubuntu arm Release Build

**Comment by jyoungyun** [comment]
@gkhanna79 @mmitche This patch is ready to merge! After merging it, I will enable ubuntu 16.04 ARM CI by auto PR trigger (related issue #1595).
/cc @hqueue @hseok-oh

File: scripts/arm32_ci_script.sh [hunk 1]
```diff
@@ -200,28 +200,32 @@ function cross_build_core_setup_with_docker {
...
     fi
-
-    # Build rootfs
     __dockerCmd="sudo docker run --privileged -i --rm -v $__currentWorkingDirectory:/opt/core-setup -w /opt/core-setup $__dockerImage"
-    __buildRootfsCmd="./cross/build-rootfs.sh $__buildArch $__linuxCodeName --skipunmount"
 
-    (set +x; echo "Build RootFS for $__buildArch $__linuxCodeName")
-    $__dockerCmd $__buildRootfsCmd
-    sudo chown -R $(id -u -n) cross/rootfs
+    if [ $__skipRootFS == 0 ]; then
```
Comment (gkhanna79) [inline scripts/arm32_ci_script.sh]: Is there a scenario to still generate RootFS? If so, then how is this generated RootFS being passed to the ___buildCmd_ below as that is hard coding the RootFS folder to be the one corresponding to Docker image?

Comment (jyoungyun) [inline scripts/arm32_ci_script.sh]: In case of armel, it still generates rootfs in docker image. So I fixed some code. Thank you.

**Comment by jyoungyun** [comment]
@dotnet-bot test Ubuntu16.04 arm Release Build

**Comment by jyoungyun** [comment]
@gkhanna79 This is ready to merge. Please take a look.

**Pull request merged**